### ISSUE TITLE
most_severe_accidents Widget (map) - API for accident severity

### ISSUE THREAD

**Issue by atalyaalon** [hasadna/anyway#1681]
The Widget API [most_severe_accidents](https://github.com/hasadna/anyway/blob/d31f622df16283fc949b985c9de7c3fb5d8c1469/anyway/infographics_utils.py#L280) serves the map in the FE, where severity is set by the [hebrew word for severity](https://github.com/hasadna/anyway-newsflash-infographics/blob/5c29ec370120fda2b0a28c04d1c0235097f55df8/src/components/atoms/AnywayMapIcon.tsx#L38) and sets the color of the accident on the map.

With localization - this does not work well when API is in another language, since localization changes the accident_severity text.

Decide on a standard together with the FE team (anyway-news-report-front slack channel) so localization won't affect the functionality (nowadays all accidents are in one color when API is in another language.

My suggestion - don't locallize the accident_severity field - and the english word will be used in FE to decide on the color.

Example:
The item from the API localizaed to Hebrew:
```
                    {
                        "longitude": 34.9034258831416,
                        "latitude": 31.2055158204296,
                        "accident_severity": "קטלנית",
                        "accident_timestamp": "2020-09-17 17:00:00",
                        "accident_type": "התהפכות"
                    }
```

The item from the API localizaed to English:
```
                    {
                        "longitude": 34.9034258831416,
                        "latitude": 31.2055158204296,
                        "accident_severity": "fatal",
                        "accident_timestamp": "2020-09-17 17:00:00",
                        "accident_type": "Overturned vehicle"
                    }
```

Attached - screenshot of a [FE map](https://anyway-infographics.web.app/newsflash/63994) with only one color:

<img width="541" alt="Screen Shot 2021-01-16 at 16 59 56" src="https://user-images.githubusercontent.com/20992625/104815309-5eef4600-581c-11eb-8843-438a3a9a2c05.png">

Note: right now for some reason our FE only uses english localization in API - that's why FE has english words and the map shows only one color even when Hebrew exists - this is a bug that is known and will be taken care of soon.

@ziv17 FYI

**Comment by atalyaalon** [comment]
@yuvalbl  FYI

**Comment by yuvalbl** [comment]
I agree - `accident_severity` should not be localized, and data should be provided in lowercase english( `fatal`, `light` etc.)

**Comment by yuvalbl** [comment]
please provide the available options for `accident_severity`

**Comment by ziv17** [comment]
I am looking for a guideline for what fields in an infographics widget data should be translated. e.g. does `accident_severity` and  `accident_type` has same (translating) behavior?

**Comment by atalyaalon** [comment]
So it depends on the widget - specifically on the map we don't have labels on the map markers - so no additional information is shown but the color of the widget (which is influenced from severity)
In the future we might add details on hover - when we'll start embedding widgets.

**Comment by ziv17** [comment]
I don't understand your answer...

**Comment by ahmedyusef9** [comment]
As I see in Postgres about the types of accident severity,
I add **accident_severity_type** column.
| accident_severity | accident_severity_hebrew | accident_severity_type |
| --- | --- | --- |
| 0 | null  | None |
| 1 | קלה | light |
| 2 | קטלנית | fatal |
| 3 | קשה | sever |
